manticore/ethereum.py

```diff
@@ -27,6 +27,12 @@ class EthereumError(ManticoreError):
     pass
 
 
+class DependencyError(EthereumError):
+    def __init__(self, lib_names):
+        super(DependencyError, self).__init__("You must pre-load and provide libraries addresses{ libname:address, ...} for %r" % lib_names)
```

Comment:
space in wrong position: `libraries addresses{ libname:address, ...}` -> `libraries addresses {libname:address, ...}`